src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -72,8 +72,20 @@ declare const __LOADER__IS_LAZY__: any;
     });
   }
 
+  function sdkIsLoaded() {
+    const __sentry = _window.__SENTRY__;
+    // If there is a global __SENTRY__ that means that in any of the callbacks init() was already invoked
+    return !!(
+      __sentry &&
+      __sentry.hub &&
+      __sentry.hub.getClient &&
+      __sentry.hub.getClient()
+    );
+  }
+
   function injectSdk(callbacks) {
-    if (injected) {
+    // If the SDK has been loaded in the meanwhile, we want to skip injecting it again
+    if (injected || sdkIsLoaded()) {
```

Comment:
Is it possible for customers to want to have multiple SDKs loaded this way?  (both loader and from bundle, possible different DSNs?)